Sources/App/Models/Package.swift

```diff
@@ -131,6 +131,14 @@ extension Package {
 }
 
 
+extension Package {
+    enum ProductType: String, Codable, CaseIterable {
+        case executable
+        case library
+        case plugin
+    }
+}
```

Comment:
I think we should move this to `ProductTypeSearchFilter`. It's only used there and we've got then closely related `Product.ProductType` at the `Model` level which this partly "overlays".